composer.json

```diff
@@ -50,10 +57,7 @@
         ]
     },
     "config": {
-        "bin-dir": "bin",
-        "platform": {
-            "php": "5.3.9"
```

Comment:
Same here. I'll restore it as is zhen it Zillow be ready. But this PR will require Symfony 3.1 anyway.